mscore/pm.h

```diff
@@ -24,6 +24,7 @@
 #include "config.h"
 #include "mididriver.h"
 
+#include <QtGlobal>
```

Comment:
All.h is automatically included everywhere. No need to mention it here and it should build without this line.